### ISSUE TITLE
[cli] CLI writes extraneous log messages on environment addition

### ISSUE THREAD

**Issue by tfield** [craftercms/craftercms#6254]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [ ] The issue is in the develop branch
- [X] The issue is in the latest released 4.1.x
- [ ] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

When using the CLI to add an environment on macOS, 3 SLF4J configuration messages appear in the terminal. These should not be there as they are disconcerting to the user.

### Steps to reproduce

Steps:

1. Open a terminal
2. Run the CLI to add an environment
3. Observe the log messages

### Relevant log output

```shell
/Volumes/Projects/foo/crafter bin/crafter-cli add-environment -e local -u http://localhost:8080 --token
Enter value for --token (The access token for authentication): 
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Error authenticating user, please check your credentials
/Volumes/Projects/foo/crafter bin/crafter-cli add-environment -e local -u http://localhost:8080 --token
Enter value for --token (The access token for authentication): 
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Authenticated as Tony Field (tony)
Environment added
/Volumes/Projects/foo/crafter
```

**Comment by phuongnq** [comment]
Fixed along with https://github.com/craftercms/cli/pull/8

**Comment by Euquimides** [comment]
Hi! Updated and rechecked this issue. Fix verified for Crafter v4.2.0 both CE and EE. Thank you! Closing this ticket.

**Comment by Euquimides** [comment]
Hi @sumerjabri ! I noticed that we tested this for 4.2.0, but didn't merged the fix into 4.1.x. Should we add this fix into 4.1.x?

**Comment by sumerjabri** [comment]
@Euquimides, this was scheduled for 4.2 only. The only reason to backport is if it's a real requirement for your work.
